### PR TITLE
doc: fix a couple of typos

### PR DESCRIPTION
**Pull request by Rjerk**
Signed-off-by: Liu Lan <liulan_yewu@cmss.chinamobile.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/dev/dashboard/ui_goals.rst [hunk 1]
```diff
@@ -47,13 +47,13 @@ ______________
...
    recent for it to be relevant and acted upon with confidence. In addition, the age of the data should
-   be shown as an age (e.g. 20s ago) rather than UTC timestamps to make it more immediately consumable by
+   be shown as age (e.g. 20s ago) rather than UTC timestamps to make it more immediately consumable by
    the Administrator.
 #. **Automate through workflows**. If the admin has to follow a 'recipe' to perform a task, the goal of
    the dashboard UI should be to implement the flow.
 #. **Provide a natural next step**. The UI **is** the *expert system*, so instead of expecting the user
-   to know where to they go next, the UI should lead them. This means linking components together to
-   establish a flow, and deeper integration between the alertmanager implementation and the dashboard
+   to know where should they go next, the UI should lead them. This means linking components together to
```
Comment (tchaikov) [inline doc/dev/dashboard/ui_goals.rst:55]:
```suggestion
   to know where they go next, the UI should lead them. This means linking components together to
```

**Review by batrick — APPROVED** [review]

**Comment by tchaikov** [comment]
```
__________________ ERROR collecting services/orchestrator.py ___________________
services/orchestrator.py:13: in <module>
    from .. import mgr
__init__.py:55: in <module>
    from .module import Module, StandbyModule  # noqa: F401
module.py:24: in <module>
    from .grafana import push_local_dashboards
grafana.py:9: in <module>
    import requests
.tox/py3/lib/python3.6/site-packages/requests/__init__.py:95: in <module>
    from urllib3.contrib import pyopenssl
.tox/py3/lib/python3.6/site-packages/urllib3/contrib/pyopenssl.py:46: in <module>
    import OpenSSL.SSL
.tox/py3/lib/python3.6/site-packages/OpenSSL/__init__.py:8: in <module>
    from OpenSSL import crypto, SSL
.tox/py3/lib/python3.6/site-packages/OpenSSL/SSL.py:673: in <module>
    _lib.Cryptography_HAS_TLSEXT_HOSTNAME, "SNI not available"
E   AttributeError: module 'lib' has no attribute 'Cryptography_HAS_TLSEXT_HOSTNAME'
```

this is a new one. looks like a bug in `pyOpenSSL`.

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by Rjerk** [comment]
jenkins test make check

**Comment by Rjerk** [comment]
jenkins retest this please

**Pull request merged**